CMakeLists.txt

```diff
@@ -83,9 +83,12 @@ target_link_libraries(videoio)
 add_executable(deepseg
   app/deepseg.cc
   app/background.cc
+  app/blend.cc
   app/calcmask.cc
   app/utils.cc
 )
+# special compile options for blend.cc to use SIMD optimiser
+set_source_files_properties(app/blend.cc PROPERTIES COMPILE_OPTIONS "-march=native;-O9")
```

Comment:
Why not for everything?

Reply:
I'm not sure.. I guess that if one module is targetted / optimised for the current CPU, then it may as well all be. Initially I only wanted to mess with / break one specific part of the code. I was also thinking about compiling run-time swappable implementations for different targets (this gets talked about a bit in the GCC manuals when using intrinsics) - that is probably over-engineering.

Reply:
I think as we are compiling all 3rd party deps ourselves we may as well compile everything for -march=native. Let's split this into another PR, so we could merge this one for now.

Having run-time swappable implementations is probably overkill for us right now. Let's handle this separately.